pkg/internalapis/cnsvolumeoperationrequest/cnsvolumeoperationrequest.go

```diff
@@ -120,22 +107,124 @@ func InitVolumeOperationRequestInterface(ctx context.Context) (VolumeOperationRe
 // Returns an error if any error is encountered while attempting to
 // read the previously persisted information from the API server.
 // If no previous information has been persisted, it returns an empty object.
-func (or *operationRequestStoreOnETCD) GetRequestDetails(ctx context.Context, name string) (*VolumeOperationRequestDetails, error) {
+func (or *operationRequestStore) GetRequestDetails(ctx context.Context, name string) (*VolumeOperationRequestDetails, error) {
 	log := logger.GetLogger(ctx)
-	// TODO: Implement GetRequestDetails for operationRequestStoreOnETCD
-	err := fmt.Errorf("GetRequestDetails not implemented for operationRequestStoreOnETCD")
-	log.Error(err)
-	return nil, err
+	namespacedName := client.ObjectKey{Name: name, Namespace: csiconfig.DefaultCSINamespace}
```

Comment:
Can we name `namespacedName` to `instanceKey`?

Reply:
done